src/components/Editor.vue

```diff
@@ -232,7 +232,6 @@ export default defineComponent({
 		const ydoc = new Doc()
 		const awareness = new Awareness(ydoc)
 		// Wrap the connection in an object so we can hand it to the Mention extension as a ref.
-		const { connection } = provideConnection()
```

Comment:
Seems like the comment above the moved line belonged to it, so move it along with it?